### ISSUE TITLE
Use reshape instead of view

### ISSUE THREAD

**Issue by vincentqb** [pytorch/audio#571]
Following #409, view is used instead of reshape. When running the [tutorial](https://github.com/pytorch/tutorials/blob/master/beginner_source/audio_preprocessing_tutorial.py), the following error occurs as mentioned [here](https://app.circleci.com/pipelines/github/pytorch/tutorials/1788/workflows/47f00fbd-88ed-486e-9edb-7de32bfb1058/jobs/30647):
```
Apr 22 18:46:35 Traceback (most recent call last):
Apr 22 18:46:35   File "/opt/conda/lib/python3.6/site-packages/sphinx_gallery/gen_rst.py", line 394, in _memory_usage
Apr 22 18:46:35     out = func()
Apr 22 18:46:35   File "/opt/conda/lib/python3.6/site-packages/sphinx_gallery/gen_rst.py", line 382, in __call__
Apr 22 18:46:35     exec(self.code, self.globals)
Apr 22 18:46:35   File "/var/lib/jenkins/workspace/beginner_source/audio_preprocessing_tutorial.py", line 216, in <module>
Apr 22 18:46:35     computed = torchaudio.functional.compute_deltas(specgram, win_length=3)
Apr 22 18:46:35   File "/opt/conda/lib/python3.6/site-packages/torchaudio/functional.py", line 1304, in compute_deltas
Apr 22 18:46:35     specgram = specgram.view(1, -1, shape[-1])
Apr 22 18:46:35 RuntimeError: view size is not compatible with input tensor's size and stride (at least one dimension spans across two contiguous subspaces). Use .reshape(...) instead.
```

Workaround: 
* add `specgram = specgram.contiguous()`, see pytorch/tutorials#963

Proposed solution:
* revert #409

**Comment by vincentqb** [comment]
cc @cpuhrsch for [comment](https://github.com/pytorch/audio/pull/391#discussion_r366004960)

**Comment by vincentqb** [comment]
Closed by #594
